addons/Popochiu/Engine/Templates/CharacterTemplate.gd

```diff
@@ -5,6 +5,9 @@ const Data := preload('CharacterStateTemplate.gd')
 
 var state: Data = null
 
+# turns anti-glide animation on.
+# will not work if call method track in animation player is not calling update_position every frame
+export var anti_glide_animation: bool = false
```

Comment:
Must be in `PopochiuCharacter.gd`, which is the class from where all characters inherit from.

This script is only the one used as the template for developers to write their logic.